Calibration/IsolatedParticles/python/hcalHBHEMuonSim_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+HcalHBHEMuonSimAnalyzer = cms.EDAnalyzer("HcalHBHEMuonSimAnalyzer",
+                                         ModuleLabel    = cms.string('g4SimHits'),
+                                         EBCollection   = cms.string('EcalHitsEB'),
+                                         EECollection   = cms.string('EcalHitsEE'),
+                                         HCCollection   = cms.string('HcalHits'),
+                                         Verbosity      = cms.untracked.int32(0),
+                                         MaxDepth       = cms.untracked.int32(4),
+                                         EtaMax         = cms.untracked.double(3.0),
+                                         TimeMinCutECAL = cms.untracked.double(-500.0),
+                                         TimeMaxCutECAL = cms.untracked.double(500.0),
+                                         TimeMinCutHCAL = cms.untracked.double(-500.0),
+                                         TimeMaxCutHCAL = cms.untracked.double(500.0),
+                                         )
```

Comment:
Why is this file still here?
If you rename the description in [fillDescriptions](https://github.com/cms-sw/cmssw/pull/18125/files#diff-640e2b52ddd6ff2c9da379584658c8c5R452) from `"hcalHBHEMuonSim"` to `"HcalHBHEMuonSimAnalyzer"` you can drop this file because it will be automatically created.